board/recalbox/fsoverlay/etc/init.d/S02splash

```diff
@@ -51,7 +51,15 @@ do_start ()
             dbus-send --print-reply=literal --session --dest=org.mpris.MediaPlayer2.omxplayer /org/mpris/MediaPlayer2 org.mpris.MediaPlayer2.Player.Action int32:15 >/dev/null
         fi
     fi
-    fbv -f -i /recalbox/system/resources/splash/logo-version.png
+
+    # on some sytems, fb0 is not immediatly loaded, so, keep a chance by waiting a bit
+    N=0
+    while ! test -e /dev/fb0 -o $N -gt 15
+    do
+	sleep 1
+	let N++
```

Comment:
recalbox-part.sh replace the fstab, need to change the sed in https://github.com/recalbox/recalbox-rescue/blob/rb-4/sdcontent/os/recalboxOS-rpi3/partition_setup.sh before merging (i'll do)